src/test/java/athena/task/TaskTest.java

```diff
@@ -0,0 +1,51 @@
+package athena.task;
+
+import athena.Importance;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class TaskTest {
+    private ArrayList<LocalDate> expectedDates;
+
+    @BeforeEach
+    public void setUp() {
+        setupExpectedDates();
+    }
+
+    private void setupExpectedDates() {
+        expectedDates = new ArrayList<>();
+        LocalDate startDate = LocalDate.now();
+        for (int i = 0; i < 10; i++) {
+            expectedDates.add(startDate.plusWeeks(i));
+        }
+    }
+
+    /**
+     * Ensure repeating dates have been set correctly.
+     */
+    @Test
+    void testSetRecurrence_today_returnsTrue() {
```

Comment:
I think it is unclear to say `today`, since `today` means only `TODAY` in our program? Something like`thisDayOfWeek` instead?